src/components/common/Datepicker.jsx

```diff
@@ -0,0 +1,142 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import { Input } from '@chakra-ui/react';
+import Separator from '../../assets/icons/Datepicker_Separator.svg';
+
+/**
+ * Datepicker 컴포넌트
+ *
+ * Props:
+ * - register: react-hook-form의 register 함수
+ * - min: 최소 날짜 (YYYY-MM-DD 형식)
+ * - max: 최대 날짜 (YYYY-MM-DD 형식)
+ * - required: 필수 입력 여부
+ * - disabled: 비활성화 여부
+ */
+
+const Datepicker = ({
+  register,
+  min,
+  max,
+  required = false,
+  disabled = false,
+}) => {
+  const [startDate, setStartDate] = useState('');
+  const [endDate, setEndDate] = useState('');
+
+  // 시작일 변경 → 종료일 최소값 설정
+  const handleStartDateChange = (e) => {
+    const value = e.target.value;
+    setStartDate(value);
+  };
+
+  // 종료일 변경 → 시작일 최대값 설정
+  const handleEndDateChange = (e) => {
+    const value = e.target.value;
+    setEndDate(value);
+  };
+
+  return (
+    <SContainer>
+      <SDateInputWrapper>
+        <SDateInput
+          as={Input}
```

Comment:
그러면 이렇게 `as`를 지정하지 않아도 됩니다.
```suggestion
        <SDateInput
```